src/Todoist.Net/Models/CompletedItem.cs

```diff
@@ -90,5 +90,14 @@ public class CompletedItem
         /// </value>
         [JsonProperty("user_id")]
         public long UserId { get; internal set; }
+
+        /// <summary>
+        /// Gets the full item object.
+        /// </summary>
+        /// <value>
+        /// The full item object.
+        /// </value>
+        [JsonProperty("item_object")]
+        public Item ItemObject { get; internal set; }
```

Comment:
Please add a remark for the property that it's filled only when...

Reply:
Done 👍